src/PublicApiAnalyzers/UnitTests/DeclarePublicAPIAnalyzerTestsBase.cs

```diff
@@ -917,19 +991,24 @@ public class C
 C.Property.set -> void
 C.Method() -> void
 ";
-            var shippedFilePath = Path.Combine(tempPath, DeclarePublicApiAnalyzer.ShippedFileName);
+            var shippedFilePath = Path.Combine(tempPath, ShippedFileName);
 
             string unshippedText = $@"";
-            var unshippedFilePath = Path.Combine(tempPath, DeclarePublicApiAnalyzer.UnshippedFileName);
+            var unshippedFilePath = Path.Combine(tempPath, UnshippedFileName);
 
             await VerifyCSharpAsync(source, shippedText, unshippedText, shippedFilePath, unshippedFilePath,
                 // <%TEMP_PATH%>\PublicAPI.Shipped.txt(7,1): warning RS0017: Symbol 'C.Method() -> void' is part of the declared API, but is either not public or could not be found
-                GetAdditionalFileResultAt(7, 1, shippedFilePath, DeclarePublicApiAnalyzer.RemoveDeletedApiRule, "C.Method() -> void"));
+                GetAdditionalFileResultAt(7, 1, shippedFilePath, RemoveDeletedApiRule, "C.Method() -> void"));
         }
 
         [Fact]
         public async Task TypeForwardsAreProcessed1Async()
         {
+            if (IsInternalTest)
```

Comment:
Testing internal versions of this would require setting up a complicated test project with IVT, and I don't think it's worth the code coverage.